openshift/template.yaml

```diff
@@ -12,7 +12,7 @@ objects:
     selector:
       matchLabels:
         app: assisted-events-scrape
-    replicas: ${{REPLICAS_COUNT}}
+    replicas: ${REPLICAS_COUNT}
```

Comment:
Isn't ``${{REPLICAS_COUNT}}`` required for it to be a numeral field and not a string?